util/cmd/cmd.go

```diff
@@ -91,3 +94,12 @@ func ParseLabels(labelSpec interface{}) (map[string]string, error) {
 	}
 	return labels, nil
 }
+
+// SetGLogLevel set the glog level for the k8s go-client
+// this is taken from argoproj/pkg but uses v2 of klog here
+// to be compatible with k8s clients v0.19.x and above
+func SetGLogLevel(glogLevel int) {
```

Comment:
minor - could go into own file

Reply:
Sounds good - i'll scoot it into a separate file real quick